add network connect|disconnect compat endpoints (#8391)
containers/podman: docs/source/markdown/podman-network-connect.1.md

% podman-network-connect(1)

## NAME
podman\-network\-connect - Connect a container to a network

## SYNOPSIS
**podman network connect** [*options*] network container

## DESCRIPTION

> Connects a container to a network. You can connect a container by name or by ID. Once connected, the container can communicate with other containers

Reviewer: Please don't use the word "You" in a man page.  
```suggestion
Connects a container to a network.  A container can be connected to the network by name or by ID. Once connected, the container can communicate with other containers
```

in the same network.

> To verify what networks a given container is connected to, use the `podman network inspect` command. Use `podman network disconnect` to remove a container from the network.

Reviewer 2: `podman network inspect` shows nothing about what networks a container is connected to. `podman container inspect` on a container would do that.

## OPTIONS
#### **--alias**

> Add network-scoped alias for the container.  You may specify multiple *--alias* inputs.

Reviewer 2: This doesn't make it clear what an alias is

> Add network-scoped alias for the container.  You may specify multiple *--alias* inputs.

Reviewer: You
```suggestion
Add network-scoped alias for the container.  Multiple *--alias* options may be specified as inputs.
```

## EXAMPLE

Connect a container named *web* to a network named *test*
```

> $ podman network connect test web

Reviewer 3: ```suggestion
podman network connect test web
```

```

Connect a container name *web* to a network named *test* with two aliases: web1 and web2
```

> $ podman network connect --alias web1 --alias web2 test web

Reviewer 3: ```suggestion
podman network connect --alias web1 --alias web2 test web
```
